gems/bundled_gems

```diff
@@ -19,4 +19,4 @@ matrix          0.4.2   https://github.com/ruby/matrix
 prime           0.1.2   https://github.com/ruby/prime
 rbs             3.1.0   https://github.com/ruby/rbs
 typeprof        0.21.7  https://github.com/ruby/typeprof aabc019684d8b4a1ed66c2a1ca48da7bbb18dcc0
-debug           1.7.2   https://github.com/ruby/debug
+debug           1.7.2   https://github.com/ruby/debug bfd220b223c765514be6b3ae334856e89e44920a
```

Comment:
This would deserve explanation in the commit message, because on the first look, this looks to be out of the scope of this PR, while this also might really be related. Hard to guess.